tests/unit/models/transactions/test_signer_list_set.py

```diff
@@ -177,3 +177,106 @@ def test_signer_entries_signer_quorum_valid(self):
             signer_entries=_SIGNER_ENTRIES_VALID,
         )
         self.assertTrue(tx.is_valid())
+
```

Comment:
I don't see a test case for more than 16 signers, but less than 32. Maybe I missed it? Otherwise, could you add it? (Should pass, just want to make sure we're testing the new functionality as well :) )

Reply:
Ah good catch, yes I'll add this.

Reply:
Just added it for above 8 and below 32 signers.